src/client/containers/NoteMenuBar.tsx

```diff
@@ -77,6 +122,16 @@ export const NoteMenuBar = () => {
           <button className="note-menu-bar-button">
             <Download size={18} onClick={downloadNotesHandler} />
           </button>
+          <div
```

Comment:
This should be a `button` like the rest of them, to retain hover and cursor effects, and instead of making the icon disappear after clicking it, just place the "Note copied!" text next to it.